stairway/src/main/java/bio/terra/stairway/queue/QueueMessageReady.java

```diff
@@ -19,16 +20,20 @@ public QueueMessageReady(String flightId) {
   }
 
   @Override
-  public void process(Stairway stairway) throws InterruptedException {
+  public boolean process(StairwayImpl stairwayImpl) throws InterruptedException {
     try {
-      boolean resumed = stairway.resume(flightId);
+      // Resumed is false if the flight is not found. We still call that a complete processing
+      // of the message and return true.
+      boolean resumed = stairwayImpl.resume(flightId);
       logger.info(
           "Stairway "
-              + stairway.getStairwayName()
+              + stairwayImpl.getStairwayName()
               + (resumed ? " resumed flight: " : " did not resume flight: ")
```

Comment:
I wouldn't have guessed from the log that "not resumed" means we didn't find the flight anymore and have given up on resuming it permanently. Maybe add some more of that to the log message?

Reply:
I re-worded the log message.